Tasks/PublishPipelineArtifactV1/task.json

```diff
@@ -0,0 +1,44 @@
+{
+    "id": "ECDC45F6-832D-4AD9-B52B-EE49E94659BE",
+    "name": "PublishPipelineArtifact",
+    "friendlyName": "Publish pipeline artifact",
+    "description": "Publish a local directory or file as a named artifact for the current pipeline",
+    "helpUrl": "https://docs.microsoft.com/azure/devops/pipelines/tasks/utility/publish-pipeline-artifact",
+    "helpMarkDown": "Publish a local directory or file as a named artifact for the current pipeline.",
+    "category": "Utility",
+    "author": "Microsoft Corporation",
+    "version": {
+        "Major": 1,
+        "Minor": 0,
+        "Patch": 0
+    },
+    "groups": [],
+    "demands": [],
+    "preview": true,
+    "minimumAgentVersion": "2.140.1",
+    "inputs": [
+        {
+            "name": "artifactName",
+            "type": "string",
+            "label": "The name of this artifact",
+            "defaultValue": "drop",
+            "required": true,
+            "helpMarkDown": "The name of this artifact."
+        },
+        {
+            "name": "targetPath",
+            "type": "filePath",
+            "label": "Path to publish",
```

Comment:
Reword: `File or directory path`

Note: I think this is clearer than the current label, but open to a better suggestion